server/src/main/java/org/apache/uniffle/server/buffer/ShuffleBufferManager.java

```diff
@@ -165,7 +165,9 @@ public ShuffleDataResult getShuffleData(
 
   void flushSingleBufferIfNecessary(ShuffleBuffer buffer, String appId,
       int shuffleId, int startPartition, int endPartition) {
-    if (this.bufferFlushEnabled && buffer.getSize() >= this.bufferFlushThreshold) {
+    // when single buffer flush is triggered, the buffer size should reach
```

Comment:
When we use multistorage and trigger single buffer flush, the buffer  size should be bigger than `rss.server.flush.cold.storage.threshold.size`, otherwise old storage will be useless.

Reply:
can also euqal to, am I right?

Reply:
done @jerqi